### PR TITLE
Fix to the error page

### PR DESCRIPTION
**Pull request by gcorvera**
Fixes an error the error page had that caused a different problem when parsing the error message from the response from khv

### CODE REVIEW

File: dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Views/Shared/RestError.cshtml [hunk 1]
```diff
@@ -29,10 +29,17 @@
     if (commonError != null)
     {
         ErrorModel deserializeObject = JsonConvert.DeserializeObject<ErrorModel>(commonError.Response.Content);
```
Comment (suryaren) [inline dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Views/Shared/RestError.cshtml:31]: >deserializeObject [](start = 19, length = 17)

nit: errorModel ..

File: dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Views/Shared/RestError.cshtml [hunk 2]
```diff
@@ -29,10 +29,17 @@
...
-        <p>@deserializeObject.Error.Message</p>
-        if (deserializeObject.Error.Details != null)
+        if (deserializeObject != null)
         {
-            <p>@deserializeObject.Error.Details</p>
+            <p>@deserializeObject.Error.Message</p>
+            if (deserializeObject.Error.Details != null)
```
Comment (suryaren) [inline dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Views/Shared/RestError.cshtml:35]: >Details [](start = 40, length = 7)

u want to check for string.IsNullorEmpty ? #WontFix

**Comment by suryaren** [comment]
:shipit:

File: dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Views/Shared/RestError.cshtml (hunk 1 above)
Comment (gcorvera) [reply dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Views/Shared/RestError.cshtml:31]: I am not sure what do you mean by this comment, could you be more specific?

---
In reply to: [117381816](https://github.com/Microsoft/healthvault-samples/pull/10#discussion_r117381816) [](ancestors = 117381816)

File: dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Views/Shared/RestError.cshtml (hunk 2 above)
Comment (gcorvera) [reply dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Views/Shared/RestError.cshtml:35]: Details is of type Object, not string.

---
In reply to: [117381871](https://github.com/Microsoft/healthvault-samples/pull/10#discussion_r117381871) [](ancestors = 117381871)

**Pull request merged**